include/net/nrf_cloud_rest.h

```diff
@@ -175,6 +175,7 @@ struct nrf_cloud_rest_pgps_request {
  *
  * @retval 0 If successful.
  *          Otherwise, a (negative) error code is returned.
+ *          See @verbatim embed:rst:inline :ref:`nrf_cloud_rest_failure` @endverbatim for details.
```

Comment:
huh, I didn't know you could do that!

Reply:
It took a lot of failed attempts with the CI to get it to work🤣